c/tskit/tables.c

```diff
@@ -9909,6 +9967,40 @@ tsk_table_collection_set_metadata_schema(tsk_table_collection_t *self,
         metadata_schema, metadata_schema_length);
 }
 
```

Comment:
Let's group these together with ``tsk_referenece_sequence_free``, so all the methods of the object are contiguous in the file (I guess with a standard banner)

Reply:
Done